CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ FEATURES:
 * MountStoreWithDB without providing a custom store works.
 * Repo is now lint compliant / GoMetaLinter with tendermint-lint integrated into CI
 * Better key output, pubkey go-amino hex bytes now output by default
+* Context now has a Logger field; logs written to the context will be recorded & returned on errors
```

Comment:
Where do these logs get returned?

Reply:
Swapped to the `app.Logger` based on discussion in SDK meeting